css/mediascreen.css

```diff
@@ -0,0 +1,111 @@
+/* HEADER STYLING */
+@media only screen and (max-width: 768px) {
+  header {
+    justify-content: space-between;
+    padding: 2rem 6rem 2rem;
+    position: sticky;
+    top: 0;
+    margin-bottom: 0;
+    border-top: 1px solid #fff;
+    -webkit-box-shadow: 0 7px 6px -1px rgba(0, 0, 0, 0.15);
+    -moz-box-shadow: 0 7px 6px -1px rgba(0, 0, 0, 0.15);
+    box-shadow: 0 7px 6px -1px rgba(0, 0, 0, 0.15);
+  }
+
+  header nav img {
+    display: none;
+  }
+
+  header nav ul.desktop-nav {
+    display: flex;
+    gap: 1rem;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  header nav ul.desktop-nav li {
+    list-style-type: none;
+  }
+
+  header nav ul.desktop-nav li a {
+    font-weight: 500;
+    letter-spacing: 0;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  header nav ul.desktop-nav li a:visited {
+    color: #344563;
+  }
+
+  header nav ul.desktop-nav li a:hover {
+    color: #6070ff;
+    text-decoration: underline;
+    text-decoration-color: #6070ff;
+  }
+
+  header nav ul.desktop-nav li a:active {
+    color: hsl(218, 31%, 30%);
+  }
+
+  section.intro-section {
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    background-image: url(../assets/headerbg.svg);
+    max-width: 100%;
+    max-height: 100vh;
+  }
+
+  section.intro-section article {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    align-items: left;
+    justify-content: center;
+    margin: 10rem 10rem 0;
+    max-width: 80%;
+    min-width: 700px;
+  }
+
+  section.intro-section div.contact-me {
+    padding: 0;
+    margin: 0 10rem 5rem;
+  }
+
+  section.project-section {
+    display: grid;
+    grid-template-columns: auto;
+    background-color: #e5e5e5;
+    align-items: center;
+    min-height: 100vh;
+  }
+
+  section.project-section article.project-card {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-evenly;
+    background: #fff;
+    min-height: 90vh;
+    margin: 2.5rem 3rem;
+  }
+
+  section.project-section article.project-card:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  section.project-section article.project-card img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    margin: auto;
+    border-radius: 8px;
+    flex: 1;
+  }
+
+  section.project-section article.project-card div.project_info {
```

Comment:
I would suggest using class name attributes to select some of these elements so as to make it easier to read